### PR TITLE
Cybereason: add new format to handle objects from API

### PR DESCRIPTION
**Pull request by squioc**
- Rename existing malop format as malop-cef format
- add new format to handle objects collected from the API of the Cybereason Platform

### CODE REVIEW

**Comment by PierrickV** [comment]
Would it be possible to use 1 format for JSON & CEF ? :thinking: 

I am afraid that this risk to be hard to maintain

**Review by PierrickV — DISMISSED** [review]

File: CybeReason/malop-cef/_meta/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name: Cybereason MalOp CEF
+slug: cybereason-malop-cef
+
+description: >-
+  MalOps, or Malicious Operations, are a set of malicious operations.
+
+  This format handles MalOps activities forwarded through syslog-CEF.
+
+data_sources:
+  Application logs: Cybereason MalOps platform provides activities of MalOps
```
[the remark this answers is not in the document]
Comment (squioc) [reply CybeReason/malop-cef/_meta/manifest.yml:11]: It's the best datasource that match the nature of MalOps

File: CybeReason/malop-json/_meta/smart-descriptions.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        {
+          "field": "cybereason.malop.id"
+        },
+        {
+          "field": "host.name"
+        }
+      ]
+    },
+    {
+      "value": "Detection {cybereason.malop.id}: user {user.domain}\\{user.name} affected by malicious activities.",
```
[the remark this answers is not in the document]
Comment (squioc) [reply CybeReason/malop-json/_meta/smart-descriptions.json]: Fixed

File: CybeReason/malop-json/_meta/smart-descriptions.json [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        {
+          "field": "cybereason.malop.root_cause.type"
+        },
+        {
+          "field": "cybereason.malop.root_cause.name"
+        }
+      ]
+    },
+    {
+      "value": "Detection {cybereason.malop.id}: Malicious activities detected on {host.name}.",
```
[the remark this answers is not in the document]
Comment (squioc) [reply CybeReason/malop-json/_meta/smart-descriptions.json]: Fixed

File: CybeReason/malop-json/_meta/manifest.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+name: Cybereason MalOp
+slug: cybereason-malop-json
+
+description: >-
+  Cybereason detect and prevent attack targeting endpoints.
+
+  All detections are gathered in MalOps. This format ingests new detections with suspicions and evidences in JSON representation.
+
+data_sources:
+  Application logs: Cybereason MalOps platform provides detections
```
[the remark this answers is not in the document]
Comment (squioc) [reply CybeReason/malop-json/_meta/manifest.yml]: Completed

**Review by PierrickV — APPROVED** [review]

**Pull request merged**